library/pod-security-policy/README.md

```diff
@@ -15,9 +15,8 @@ An adminstrator can control the following by setting the field in PSP or by depl
 | Usage of volume types                             | `volumes`                                                                   | [volumes](volumes)                                       |
 | Usage of the host filesystem                      | `allowedHostPaths`                                                          | [host-filesystem](host-filesystem)                       |
 | White list of Flexvolume drivers                  | `allowedFlexVolumes`                                                        | [flexvolume-drivers](flexvolume-drivers)                 |
-| Allocating an FSGroup that owns the pod's volumes | `fsGroup`                                                                   | [fsgroup](fsgroup)<sup>\*</sup>                          |
 | Requiring the use of a read only root file system | `readOnlyRootFilesystem`                                                    | [read-only-root-filesystem](read-only-root-filesystem)   |
-| The user and group IDs of the container           | `runAsUser`, `runAsGroup`, `supplementalGroups`                             | [users](users)
+| The user and group IDs of the container           | `runAsUser`, `runAsGroup`, `supplementalGroups`, `fsgroup`                             | [users](users)<sup>\*</sup>
```

Comment:
What is this bit for? : `<sup>\*</sup>`

Also, needs DCO

Reply:
I copied it over from the old fsgroup description. It's the asterix that links to `For PSP rules that apply default value or mutations, Gatekeeper v3 currently cannot apply mutation.`